glue/viewers/common/qt/data_slice_widget.py

```diff
@@ -94,18 +102,50 @@ def __init__(self, label='', pix2world=None, lo=0, hi=10,
         slider.button_next.clicked.connect(nonpartial(self._browse_slice, 'next'))
         slider.button_last.clicked.connect(nonpartial(self._browse_slice, 'last'))
 
+        slider.checkbox_world.toggled.connect(nonpartial(self.set_label_from_slider))
+
+        if world is None:
+            self.use_world = False
+            slider.checkbox_world.hide()
+        else:
+            self.use_world = not world_warning
+
         layout.addWidget(slider)
 
         self.setLayout(layout)
 
         self._ui_label = label
-        self._ui_slider = slider
         self._ui_mode = mode
         self._update_mode()
         self._frozen = False
 
         self._play_speed = 0
 
+        self.set_label_from_slider()
+
+    def set_label_from_slider(self):
+        value = self._ui_slider.slider.value()
+        if self.use_world:
+            text = str(self._world[value])
+            if self._world_warning:
+                self._ui_slider.label_warning.show()
+            else:
+                self._ui_slider.label_warning.hide()
+        else:
+            text = str(value)
+            self._ui_slider.label_warning.hide()
+        self._ui_slider.label.setText(text)
+
+    def set_slider_from_label(self):
+        text = self._ui_slider.label.text()
+        if self.use_world:
+            # Don't want to assume world is sorted, pick closest value
+            value = np.argmin(np.abs(self._world - float(text)))
+            self._ui_slider.label.setText(str(self._world[value]))
+        else:
```

Comment:
@astrofrog Can you tell me what does this value mean here?
`value = np.argmin(np.abs(self._world - float(text)))` and why there is a 
`self._world-float(text)`